packages/flutter/lib/src/material/ink_splash.dart

```diff
@@ -80,10 +80,12 @@ class InkSplash extends InkFeature {
     Color color,
     bool containedInkWell: false,
     RectCallback rectCallback,
+    BorderRadius borderRadius,
     double radius,
     VoidCallback onRemoved,
   }) : _position = position,
        _color = color,
+        _borderRadius = borderRadius ?? BorderRadius.zero,
```

Comment:
this should probably just be a default value for the parameter itself (and then assert it's not null)